### PR TITLE
Fix "Can't read create_properties of undefined"

### PR DESCRIPTION
**Pull request by HenriChabrand**
## Description

Fix of issue preventing user to use the "Create Database Page" command

## Type of change

- Bug fix / improvement for my extension

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

**Comment by HenriChabrand** [comment]
Hello @pernielsentikaer 👋 
I've introduced a bug in the past released preventing user to use the `Create Database Page` command – here's the fix.
Thanks

**Comment by pernielsentikaer** [comment]
Hi @HenriChabrand 👋 

Thanks for the update, I marked it as accepted right away 😄

File: extensions/notion/src/components/CreateDatabaseForm.tsx [hunk 1]
```diff
@@ -203,7 +203,7 @@ export function CreateDatabaseForm(props: { databaseId?: string; setRefreshView?
       const loadedDatabaseView = await loadDatabaseView(databaseId);
       const loadedDatabaseViewCopy = loadedDatabaseView ? JSON.parse(JSON.stringify(loadedDatabaseView)) : {};
 
-      if (loadedDatabaseView.create_properties) {
+      if (loadedDatabaseViewCopy?.create_properties) {
```
Comment (mathieudutour) [inline extensions/notion/src/components/CreateDatabaseForm.tsx:206]: that seems weird to me. If `loadedDatabaseViewCopy` can be undefined, then `loadedDatabaseViewCopy.create_properties = [];` on line 209 will crash as well.

Shall we change `const loadedDatabaseViewCopy = loadedDatabaseView ? JSON.parse(JSON.stringify(loadedDatabaseView)) : {};` to `const loadedDatabaseViewCopy = (JSON.parse(JSON.stringify(loadedDatabaseView)) : {}) || {};` to make sure it's never falsy?

Comment (HenriChabrand) [reply extensions/notion/src/components/CreateDatabaseForm.tsx:206]: Hello @mathieudutour 👋 

I'm not sure to see in which case `loadedDatabaseViewCopy` can be undefined with this expression?
`const loadedDatabaseViewCopy = loadedDatabaseView ? JSON.parse(JSON.stringify(loadedDatabaseView)) : {}` 

It either has the value of `loadedDatabaseView`, which is for sure a `DatabaseView` object.
Or `{}`, an empty object that will be able to receive the `create_properties` attribute line 209:
`loadedDatabaseViewCopy.create_properties = [];`


If that's not the case (I'm not 100% a TypeScript expert), let me know and I'll change the code by what you suggested 😃

Comment (mathieudutour) [reply extensions/notion/src/components/CreateDatabaseForm.tsx:206]: I don't know either - but your change (adding a `?.`) is only useful if it can be.

Comment (HenriChabrand) [reply extensions/notion/src/components/CreateDatabaseForm.tsx:206]: Ok got it, so I'll remove the `.?` as it shouldn't 🙏

**Comment by HenriChabrand** [comment]
All green @mathieudutour @pernielsentikaer ✅ 
Thanks for your feedbacks 🙏

**Review by mathieudutour — APPROVED** [review]

**Pull request merged**